src/orquestra/sdk/_base/_env.py

```diff
@@ -47,12 +69,22 @@
     ORQ_RAY_SET_TASK_RESOURCES=1
 """
 
-PASSPORT_FILE_ENV = "ORQUESTRA_PASSPORT_FILE"
-"""
-Consumed by the Workflow SDK to set auth in remote contexts
-"""
-
 RAY_GLOBAL_WF_RUN_ID_ENV = "GLOBAL_WF_RUN_ID"
 """
 Used to set the workflow run ID in a Ray workflow
 """
+
+
+# ------------------------------- utilities ----------------------------------
+
+
+def _is_truthy(env_var_value: t.Optional[str]):
```

Comment:
nitpick: Cant think of any better name of that function, but for me truthy in Python is anything that I can check with if () and is evaluated as true, I.E. list with elements [1].
But maybe the function is so trivial that it is fine 🤔

Reply:
It's about truthy-ness in the context of env variables, as specified by the module name 🙃 I don't have a better idea for this name either, so I'll leave it as is; especially given it's a private function.